[2021-11-29 12:31:23,430] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2020-02-08T00:00:00+00:00 [queued]>
[2021-11-29 12:31:24,088] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2020-02-08T00:00:00+00:00 [queued]>
[2021-11-29 12:31:24,088] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 12:31:24,088] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-11-29 12:31:24,088] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 12:31:24,630] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_2> on 2020-02-08T00:00:00+00:00
[2021-11-29 12:31:24,827] {standard_task_runner.py:52} INFO - Started process 141444 to run task
[2021-11-29 12:31:24,645] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_2', '2020-02-08T00:00:00+00:00', '--job-id', '248', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmptrcdj36a', '--error-file', '/tmp/tmp8ylhz8t_']
[2021-11-29 12:31:25,249] {standard_task_runner.py:77} INFO - Job 248: Subtask task_2
[2021-11-29 12:31:27,271] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_2 2020-02-08T00:00:00+00:00 [running]> on host airflowvm
[2021-11-29 12:31:28,583] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2020-02-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-02-08T00:00:00+00:00
[2021-11-29 12:31:28,585] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-11-29 12:31:28,638] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-11-29 12:31:28,725] {subprocess.py:75} INFO - Output:
[2021-11-29 12:31:31,822] {subprocess.py:83} INFO - Command exited with return code 0
[2021-11-29 12:31:32,545] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_2, execution_date=20200208T000000, start_date=20211129T123123, end_date=20211129T123132
[2021-11-29 12:31:33,182] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-11-29 12:31:33,213] {local_task_job.py:151} INFO - Task exited with return code 0
